src/components/OSCALInventoryItem.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell, { tableCellClasses } from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import OSCALResponsibleRoles from "./OSCALResponsibleRoles";
+import OSCALImplementedComponents from "./OSCALImplementedComponents";
+import StyledTooltip from "./OSCALStyledTooltip";
+
+const StyledTableCell = styled(TableCell)(({ theme }) => ({
+  [`&.${tableCellClasses.head}`]: {
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.common.white,
+  },
+  textAlign: "left",
+  minWidth: "10em",
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  // Use hover color for even numbered rows
+  "&:nth-of-type(even)": {
+    backgroundColor: theme.palette.action.hover,
+  },
+}));
+
+export default function OSCALInventoryItem(props) {
+  if (!props.inventoryItem) {
+    return null;
+  }
+
+  return (
+    <>
+      <Grid item xs={12}>
+        <Typography variant="subtitle1" gutterBottom component="div">
+          {props.inventoryItem?.description}
+        </Typography>
+        <Typography variant="body2">{props.inventoryItem?.remarks}</Typography>
+      </Grid>
+      <Grid item xs={12}>
+        <TableContainer sx={{ maxHeight: "15em" }}>
+          <Table aria-label="Inventory Items" sx={{ height: "max-content" }}>
+            <TableHead>
+              <TableRow>
+                <StyledTableCell>Item</StyledTableCell>
+                <StyledTableCell>Value</StyledTableCell>
+                <StyledTableCell>Responsible Parties</StyledTableCell>
+                <StyledTableCell>Implemented Components</StyledTableCell>
+                <StyledTableCell>Remarks</StyledTableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {Object.values(props.inventoryItem.props).map((item) => (
```

Comment:
Would this `map` ever return more than one item? It seems like you are mapping a single item in `SystemImplementation` then trying to map that item object. That is probably why you needed to use `Object.values`

Reply:
I originally thought `inventory-items` was 0-infity, but it's 0-1, making this mapping unnecessary. Will make this change.

Reply:
https://github.com/EasyDynamics/oscal-react-library/blob/72ad88399e0d5defb60c95d6a656976984ba0c72/src/components/OSCALSystemImplementation.js#L120-L124

I understand that, I think you are accidently mapping twice. There is a single list `Inventory-Items`. Inside the list there are `Inventory-item` objects, not more lists.

Reply:
Actually wait never mind. I am getting confused by the multiple tables for each item I think. If there are multiple tables, then this would be right.